Source/Common/gdcmSwapper.txx

```diff
@@ -16,6 +16,8 @@
 
 #if defined(_MSC_VER)
 
+#include <cstring>
```

Comment:
The compiler error is on Linux, will not help.

```
2022-01-05T22:29:31.5821948Z /home/vsts/work/1/s/Modules/ThirdParty/GDCM/src/gdcm/Source/Common/gdcmSwapper.txx: In static member function 'static T gdcm::SwapperDoOp::Swap(T) [with T = float]':
2022-01-05T22:29:31.5823350Z /home/vsts/work/1/s/Modules/ThirdParty/GDCM/src/gdcm/Source/Common/gdcmSwapper.txx:180:5: error: 'memcpy' was not declared in this scope
2022-01-05T22:29:31.5824395Z      memcpy(&temp, &val, sizeof(uint32_t));
2022-01-05T22:29:31.5850093Z      ^~~~~~
```

Reply:
Doh! Thanks!